hack/aks/Makefile

```diff
@@ -77,6 +77,11 @@ swift-net-up: ## Create vnet, nodenet and podnet subnets
 	$(AZCLI) network vnet subnet create -g $(GROUP) --vnet-name $(VNET) --name nodenet --address-prefixes 10.240.0.0/16 -o none
 	$(AZCLI) network vnet subnet create -g $(GROUP) --vnet-name $(VNET) --name podnet --address-prefixes 10.241.0.0/16 -o none
 
```

Comment:
nit:  prob add a comment that podsubnet size here leads to cluster creation in vnetscale mode?

Reply:
Not needed out here as this will change and subnet size is only for private preview. Also folks hardly ever update this piece.